packages/sdk/src/types/runtime/actor.ts

```diff
@@ -437,6 +439,59 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
         return null;
     }
 
+    /**
+     * Starts playing a preloaded sound.
+     * @param soundAssetId Name of sound asset preloaded using AssetManager.
+     * @param options Adjustments to pitch and volume, and other characteristics.
+     * @param startTimeOffset How many seconds to offset into the sound
+     */
+    public playSound(
+        soundAssetId: string, options: SetSoundStateOptions,
+        startTimeOffset?: number): ForwardPromise<SoundInstance> {
+
+        return createForwardPromise(new SoundInstance("SoundAssetHandle"),

```

Comment:
>createForwardPromise [](start = 15, length = 20)

All the work this method is doing should be done in the internal Context class. We don't send messages from here, but always send them from methods on internal/context.ts.